src/inventory.go

```diff
@@ -1,11 +1,11 @@
 package main
 
 import (
+	"github.com/newrelic/infra-integrations-sdk/data/inventory"
 	"regexp"
 	"strings"
 
 	"github.com/newrelic/infra-integrations-sdk/log"
-	"github.com/newrelic/infra-integrations-sdk/sdk"
 )
 
 func getRawInventory(config map[string]string, metrics map[string]interface{}) map[string]interface{} {
```

Comment:
minor: `inventory` variable needs to be renamed since we have an imported pkg called `inventory`.